zproject/settings.py

```diff
@@ -149,6 +149,7 @@ def get_secret(key):
                     'DBX_APNS_CERT_FILE': None,
                     'EXTRA_INSTALLED_APPS': [],
                     'DEFAULT_NEW_REALM_STREAMS': ["social", "general", "zulip"],
+                    'TERMS_OF_SERVICE': False,
```

Comment:
Do you think it'd be clearer to have this default to `None` rather than `False`?  Assuming both work, it'd have better type consistency (it'd be an Optional[text_type], not a Union[bool, text_type]))